mkdocs.yml

```diff
@@ -74,8 +79,21 @@ nav:
       - Local Runs: development.md
       - Custom Resources: development-crds.md
   - OpenAPI Definition: openapi.md
+=======
+  - Integrations:
+      - Integrating with CI/CD: testkube-automation.md
+      - Integrating with Slack: slack-integration.md
+  - Authentication:
+      - OAuth for UI: oauth.md
+      - OpenAPI Definition: openapi.md
+  - Contributing:    
+      - Contributing: contributing.md
+      - Development: development.md
+  - Architecture: architecture.md
+>>>>>>> Stashed changes
```

Comment:
Is this line ">>>>>>> Stashed changes" supposed to be here ?
Otherwise, approved ✅

Reply:
Fixed.